keyboards/mokey/mokey12x2/mokey12x2.c

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2021-2022 Alin M Elena <alinm.elena@gmail.com>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "quantum.h"
+
+#if defined(ENCODER_ENABLE)
+
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+    if (!encoder_update_kb(index, clockwise)) {
```

Comment:
FFS, for the last time, apply the suggestion correctly

```suggestion
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) {
```